Assets/Scripts/GameManager.cs

```diff
@@ -7,6 +7,13 @@ public class GameManager : MonoBehaviour
     private float nextActionTime = 0.0f;
     private float period = 1.0f;
 
+    public float secondsLeft = 30.0f;
+
+    public float getSecondsLeft()
+    {
+        return secondsLeft;
+    }
+

```

Comment:
tbh we don't need this method, but thought I would add it in case we decided to make `secondsLeft` private